sfx.js

```diff
@@ -52,9 +54,10 @@ SfxPlugin.prototype.disable = function() {
   if (this.harvestPlugin) this.harvestPlugin.removeListener('harvested', this.onHarvested);
 };
 
-SfxPlugin.prototype.play = function(name, loop, url) {
+SfxPlugin.prototype.play = function(name, loop, volume, url) {
```

Comment:
I'm guessing the conflict is here? If so can you just merge it and fix the conflict because this change was intentional